alerters/alerters-plugins/alerters-kafka/src/main/java/org/hawkular/alerter/kafka/KafkaAlerter.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerter.kafka;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+
+import org.hawkular.alerts.alerters.api.Alerter;
+import org.hawkular.alerts.alerters.api.AlerterPlugin;
+import org.hawkular.alerts.api.model.trigger.Trigger;
+import org.hawkular.alerts.api.model.trigger.TriggerKey;
+import org.hawkular.alerts.api.services.AlertsService;
+import org.hawkular.alerts.api.services.DefinitionsService;
+import org.hawkular.alerts.api.services.DistributedEvent;
+import org.hawkular.commons.log.MsgLogger;
+import org.hawkular.commons.log.MsgLogging;
+import org.hawkular.commons.properties.HawkularProperties;
+
+/**
+ * This is the main class of the Kafka Alerter.
+ *
+ * The Kafka Alerter will listen for triggers tagged with "Kafka" tag. The Alerter will create a consumer to a Kafka Topic
+ * with the config provider into the Trigger context. The Alerter will convert Kafka records into Hawkular Data or Events
+ * and send them into the Alerting engine.
+ *
+ * The Elasticsearch Alerter uses the following conventions for trigger tags and context:
+ *
+ * <pre>
+ *
+ * - [Required]    trigger.tags["Kafka"] = "<value reserved for future uses>"
```

Comment:
oh, you are right :-) prometheus is lowercase.
Well, I would leave it as it is for now, perhaps we can modify the alerters tags in following iteration.
About Kafka most of documentation I've read to built this integration references it in first letter upper case.